index.d.ts

```diff
@@ -471,6 +499,35 @@ declare namespace Moleculer {
 		}
 	}
 
+	class Transit {
+		constructor(broker: ServiceBroker, transporter: Transporter, opts: GenericObject);
```

Comment:
I wasn't able to determine what the `opts` parameter could contain, it can stay as a `GenericObject` or a better definition can be created

Reply:
Currently there is only one optional option `maxQueueSize: Number`

Reply:
Thanks, I'll add that